client/src/components/item/EditItemWrapper.tsx

```diff
@@ -0,0 +1,79 @@
+import React, {Component} from "react";
+import {match} from "react-router";
+import {Query} from "react-apollo";
+import gql from "graphql-tag";
+import ItemEditForm from "./ItemEditForm";
+import {Header, Loader, Message} from "semantic-ui-react";
+
+interface EditItemProps {
+    match: match & EditItemParams;
+}
+
+interface EditItemParams {
+    params: { itemId: string };
+}
+
+interface EditItemState {
+    item_name: string;
+}
+
+const ITEM_QUERY = gql`
+    query getItem($itemId: Int!) {
+        item(id: $itemId) {
+            item_name
+            description
+            imageUrl
+            category
+            totalAvailable
+            maxRequestQty
+            price
+            approvalRequired
+            returnRequired
+            hidden
+            owner
+        }
+    }`;
+
+class EditItemWrapper extends Component<EditItemProps, EditItemState> {
+    constructor(props: EditItemProps) {
+        super(props);
+        this.state = {
+            item_name: ""
+        };
+    }
+
+
+    public render() {
+        const itemId: number = parseInt(this.props.match.params.itemId, 10);
+        console.log(itemId);
+        return (
+            <div>
+                <Header as="h1">Edit item</Header>
+                <Query
+                    query={ITEM_QUERY}
+                    variables={
+                        {itemId}
+                    }
+                    fetchPolicy="no-cache"
+                >
+                    {
+                        ({loading, error, data}: any) => {
+                            if (loading) {
+                                return <Loader active={true} content="Just a sec!"/>;
```

Comment:
so why does the loader appear below the footer? Can you may add a div with spacing so it doesn't look bugged?

Reply:
Yeah I can do that.  It's because Semantic UU's loader centers on the page so there's nothing above the footer

Reply:
Turns out it was an easy fix, just needed to make the loader inline (rather than centered on the screen) and centered.

New loading screen:
![New edit item loading screen](https://user-images.githubusercontent.com/5790137/60065722-77dc5780-96ca-11e9-88c7-900e1bdbb89e.png)